db/migrate/20150904181202_miq_groups_add_ldap_role.rb

```diff
@@ -0,0 +1,17 @@
+class MiqGroupsAddLdapRole < ActiveRecord::Migration
+  class MiqUserRole < ActiveRecord::Base
+  end
+
+  class MiqGroup < ActiveRecord::Base
+  end
```

Comment:
I really wish my brain would remember why we had to introduce this long ago, but I just can't remember...I distinctly remember needing to go back and add this to many classes so they would not look for STI based models, particularly so they weren't accidentally picked up from the models directory.  Maybe it is time to go back and review this.

It is definitely needed when type *is* already on the table, otherwise you'd have to declare every possible subclass.